### PR TITLE
CONSOLE-1939 Update Console docs for declarative configs

### PR DESCRIPTION
**Pull request by cajieh**
The verification of the configs is in progress.

### CODE REVIEW

File: modules/adding-a-custom-logo.adoc [hunk 1]
```diff
@@ -20,7 +20,20 @@ You can create custom branding by adding a custom logo or custom product name. Y
...
+apiVersion: operator.openshift.io/v1
+kind: ConfigMap
+metadata:
+  name: console-custom-logo
+  namespace: openshift-config
+data:
+  console-custom-logo.png: <base64-encoded_logo> ... <1>
 
+----
+<1> Provide valid base64 encoded logo
```
Comment (ahardin-rh) [inline modules/adding-a-custom-logo.adoc:35]: Provide a valid base64-encoded logo.

File: modules/customizing-the-web-console-URL.adoc [hunk 2]
```diff
@@ -15,15 +15,42 @@ You can update the web console URL, `consoleURL`, to a custom value.
...
+data:
+  tls.crt: >-
+        <base64-encoded_cert> ... <1>
+  tls.key: >-
+        <base64-encoded_key> ... <2>
+----
+<1> Provide valid tls cert
+<2> Provide valid tls key
++
+Then, edit the web console’s Operator configuration, add the following stanza to the configuration resource:
```
Comment (ahardin-rh) [inline modules/customizing-the-web-console-URL.adoc:53]: configuration. Add the following

File: modules/customizing-the-web-console-URL.adoc [hunk 3]
```diff
@@ -15,15 +15,42 @@ You can update the web console URL, `consoleURL`, to a custom value.
...
+  name: console-tls
+  namespace: openshift-config
+type: kubernetes.io/tls
+data:
+  tls.crt: >-
+        <base64-encoded_cert> ... <1>
+  tls.key: >-
+        <base64-encoded_key> ... <2>
+----
+<1> Provide valid tls cert
```
Comment (ahardin-rh) [inline modules/customizing-the-web-console-URL.adoc:50]: <1> Provide a valid TLS certificate.
<2> Provide a valid TLS key.

**Comment by ahardin-rh** [comment]
@cyril-ui-developer Just a few minor style edits from me. Thank you for this!

**Comment by cyril-ui-developer** [comment]
> @cyril-ui-developer Just a few minor style edits from me. Thank you for this!

@ahardin-rh Updated! Thanks.

**Comment by ahardin-rh** [comment]
@cyril-ui-developer Your updates look good to me! Thank you! Please let me know when the configs are verified. I assume you are working with QE for verification? If that's not the case, please let me know and I will reach out to QE for review. Thanks again!

**Comment by ahardin-rh** [comment]
@cyril-ui-developer Hey, Cyril! Any update on the status of this? Thanks!

**Comment by cyril-ui-developer** [comment]
> @cyril-ui-developer Hey, Cyril! Any update on the status of this? Thanks!

@ahardin-rh Please reach out to @spadgett. I'm on leave. Thanks

**Comment by ahardin-rh** [comment]
@spadgett PTAL. Thanks!

**Review by spadgett — APPROVED** [review]
lgtm

**Comment by ahardin-rh** [comment]
@siamaksade PTAL. Thank you!

**Comment by ahardin-rh** [comment]
Received approval from Siamak via Slack. Merging now.  Thanks, all!

**Pull request merged**